core/runtime/module_factory.hpp

```diff
@@ -54,13 +55,15 @@ namespace kagome::runtime {
      * Compile `wasm` code to `path_compiled`.
      */
     virtual CompilationOutcome<void> compile(
-        std::filesystem::path path_compiled, BufferView wasm) const = 0;
+        std::filesystem::path path_compiled,
+        BufferView wasm,
+        const RuntimeContext::ContextParams &config) const = 0;
 
     /**
      * Load compiled code from `path_compiled`.
      */
     virtual CompilationOutcome<std::shared_ptr<Module>> loadCompiled(
-        std::filesystem::path path_compiled) const = 0;
+        const kagome::parachain::PvfWorkerInputCodeParams &code_params) const = 0;
```

Comment:
`ModuleFactory` interface should not depend on `PvfWorkerInputCodeParams`, which is related to pvf.
```suggestion
        std::filesystem::path path_compiled, const RuntimeContext::ContextParams &config) const = 0;
```